src/backend/_scss/application.scss

```diff
@@ -848,10 +849,30 @@ a {
 
 .track-info {
   margin-bottom: 5px;
+  &:hover {
+    .track-name {
+      color: $light-blue;
+    }
+    .titlecolor {
+      height: 14px;
+      margin-right: 0;
+      width: 14px;
+    }
+  }
 }
 
 .room-info {
   margin-bottom: 5px;
+  &:hover {
+    .room-name {
+      color: $light-blue;
+    }
+    .titlecolor {
+      height: 14px;
+      margin-right: 0;
+      width: 14px;
```

Comment:
An alternative to setting specific widths and heights could be:
```
      transform: scale(1.16);
```
That should cause the circle to expand about its centre.

`transform:scale()` does not affect the width+height of the element (or its container) in the page, only how it is displayed.